.github/workflows/welcome-external-pr.yml

```
@@ -0,0 +1,53 @@
+name: External contribution guidelines
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+
+env:
+  ORGANIZATION: Ethereum
+  DRY_RUN: true
+
+jobs:
+  comment-external-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get organization members
+        id: get_members
+        env:
+          GH_TOKEN: ${{ secrets.READ_ORG }}
+          CONTRIBUTOR: ${{ github.event.pull_request.user.login }}
+        run: |
+          gh api graphql \
+            --raw-field organization="$ORGANIZATION" \
+            --raw-field query='
+              query($organization: String!, $cursor: String) {
+                organization(login: $organization) {
+                  membersWithRole(first: 100, after: $cursor) {
+                    pageInfo {
+                      hasNextPage,
+                      endCursor
+                    }
+                    nodes {
+                      login
+                    }
+                  }
+                }
+              }' > org_members.json
+          echo "CONTRIBUTOR_IS_ORG_MEMBER=$(jq --arg contributor $CONTRIBUTOR '.data.organization.membersWithRole | any(.nodes[].login; . == $contributor)' org_members.json)" >> $GITHUB_OUTPUT
```

Let's wrap this for readability:
```suggestion
          echo "CONTRIBUTOR_IS_ORG_MEMBER=$(
            jq \
              --arg contributor $CONTRIBUTOR \
              '.data.organization.membersWithRole | any(.nodes[].login; . == $contributor)' \
              org_members.json
          )" >> $GITHUB_OUTPUT
```